### PR TITLE
Provide names (and original names) for all Termina Field cutscenes

### PR DESCRIPTION
**Pull request by tom-overton**
I found out that you can get original cutscene names from MM3D (for those who want to recreate this, go to `romfs/scenes` and find `_info.zsi` files, *not* `_0_info.zsi` files; decompress these files and search for ".jdb"), so I figured I'd get started by picking a scene with a lot of cutscenes. I tried to provide sensible names for these, but it's actually quite hard to think of a short way to summarize a cutscene, so I'm open to suggestions if anyone has any.

### CODE REVIEW

File: assets/xml/scenes/Z2_00KEIKOKU/Z2_00KEIKOKU.xml [hunk 1]
```diff
@@ -2,6 +2,30 @@
...
+        <Cutscene Name="gTerminaFieldSkullKidDrawingStartCs" Offset="0x3A68"/> <!-- Original name is "find_rakugaki" ("rakugaki" = "graffiti") -->
+        <Cutscene Name="gTerminaFieldSkullKidDrawingEndCs" Offset="0x4208"/> <!-- Original name is "rakugaki_end" -->
+        <Cutscene Name="gTerminaFieldIntroFromWestClockTownCs" Offset="0x4568"/> <!-- Original name is "sc1st01_001valley0" -->
+        <Cutscene Name="gTerminaFieldIntroFromNorthClockTownCs" Offset="0x4858"/> <!-- Original name is "sc1st01_001valley1" -->
+        <Cutscene Name="gTerminaFieldIntroFromEastClockTownCs" Offset="0x4A68"/> <!-- Original name is "sc1st01_001valley2" -->
+        <Cutscene Name="gTerminaFieldIntroFromSouthClockTownCs" Offset="0x4C78"/> <!-- Original name is "sc1st01_001valley3" -->
+        <Cutscene Name="gTerminaFieldGiantsGatherToStopMoonCs" Offset="0xDE2C"/> <!-- Original name is "k_s_start" ("k" is probably short for "kyojin", meaning "giant") -->
+        <Cutscene Name="gTerminaFieldGiantsFailToStopMoon" Offset="0xFA3C"/> <!-- Original name is "k_s_sippai" ("failure") -->
```
Comment (AngheloAlf) [inline assets/xml/scenes/Z2_00KEIKOKU/Z2_00KEIKOKU.xml:17]:
```suggestion
        <Cutscene Name="gTerminaFieldGiantsFailToStopMoonCs" Offset="0xFA3C"/> <!-- Original name is "k_s_sippai" ("failure") -->
```

**Review by AngheloAlf — COMMENTED** [review]
one small nit

File: assets/xml/scenes/Z2_00KEIKOKU/Z2_00KEIKOKU.xml (hunk 1 above)
Comment (tom-overton) [reply assets/xml/scenes/Z2_00KEIKOKU/Z2_00KEIKOKU.xml]: Whoops, thanks for catching this one!

**Review by AngheloAlf — APPROVED** [review]

File: assets/xml/scenes/Z2_00KEIKOKU/Z2_00KEIKOKU.xml [hunk 2]
```diff
@@ -2,6 +2,30 @@
...
+        <Cutscene Name="gTerminaFieldSkullKidDrawingEndCs" Offset="0x4208"/> <!-- Original name is "rakugaki_end" -->
+        <Cutscene Name="gTerminaFieldIntroFromWestClockTownCs" Offset="0x4568"/> <!-- Original name is "sc1st01_001valley0" -->
+        <Cutscene Name="gTerminaFieldIntroFromNorthClockTownCs" Offset="0x4858"/> <!-- Original name is "sc1st01_001valley1" -->
+        <Cutscene Name="gTerminaFieldIntroFromEastClockTownCs" Offset="0x4A68"/> <!-- Original name is "sc1st01_001valley2" -->
+        <Cutscene Name="gTerminaFieldIntroFromSouthClockTownCs" Offset="0x4C78"/> <!-- Original name is "sc1st01_001valley3" -->
+        <Cutscene Name="gTerminaFieldGiantsGatherToStopMoonCs" Offset="0xDE2C"/> <!-- Original name is "k_s_start" ("k" is probably short for "kyojin", meaning "giant") -->
+        <Cutscene Name="gTerminaFieldGiantsFailToStopMoonCs" Offset="0xFA3C"/> <!-- Original name is "k_s_sippai" ("failure") -->
+        <Cutscene Name="gTerminaFieldGiantsStopMoonCs" Offset="0x1040C"/> <!-- Original name is "k_s_seikou" ("success") -->
```
[the remark this answers is not in the document]
Comment (tom-overton) [reply assets/xml/scenes/Z2_00KEIKOKU/Z2_00KEIKOKU.xml]: I did `StopMoonFailure` and `StopMoonSuccess` instead, but I see your point. Let me know if that still doesn't work for you.

**Review by isghj5 — APPROVED** [review]

**Review by engineer124 — APPROVED** [review]

**Pull request merged**